packages/components/accordion/src/accordion.stories.ts

```diff
@@ -227,6 +229,78 @@ export const Sticky: Story = {
   }
 };
 
+export const ToggleExternally: Story = {
+  render: ({ single }) => {
+    try {
+      customElements.define(
+        'accordion-toggle-example',
+        class extends LitElement {
+          alien = false;
+          dino = true;
+
+          override render(): TemplateResult {
+            return html`
+              <sl-button @click=${this.toggleDino}>Toggle 🦖</sl-button>
+              <sl-button @click=${this.toggleAlien}>Toggle 👽</sl-button>
+              <p>
+                🦖 state: ${this.dino ? 'open' : 'closed'}<br />
+                👽 state: ${this.alien ? 'open' : 'closed'}
+              </p>
+              <sl-accordion ?single=${single}>
+                <sl-accordion-item
+                  summary="🦖"
+                  .open=${this.dino}
+                  @sl-toggle=${(e: SlToggleEvent) => this.onToggle(e, 'dino')}
+                >
+                  Discovering Dinosaurs: A Prehistoric Adventure Embark on a thrilling journey back in time to the age
+                  of dinosaurs! 🌎🦕🌿🦖
+                </sl-accordion-item>
+                <sl-accordion-item summary="🏛️">
+                  Journey Through Ancient Civilizations Pack your virtual bags and travel through time to ancient Egypt,
+                  Greece, Rome, and beyond 🌍🏛️🔍🏺
+                </sl-accordion-item>
+                <sl-accordion-item
+                  summary="👽"
+                  .open=${this.alien}
+                  @sl-toggle=${(e: SlToggleEvent) => this.onToggle(e, 'alien')}
```

Comment:
Casting to `SlToggleEvent` should be unnecessary because `sl-toggle` already has that type.

Reply:
![alt text](image.png)

Reply:
It doesn't work without it...

Reply:
And this is also done in other places `@sl-toggle` is used:
<img width="672" alt="image" src="https://github.com/user-attachments/assets/043ffc14-2088-4763-bd34-9faff0e70d90">